packages/ui/src/components/CheckBox.svelte

```diff
@@ -24,19 +24,14 @@
 
   const dispatch = createEventDispatcher()
 
-  let oldChecked = checked
-
   const handleValueChanged = (event: Event): void => {
     if (readonly) {
       return
     }
     const eventTarget = event.target as HTMLInputElement
     const isChecked = eventTarget.checked
 
-    if (oldChecked !== isChecked) {
```

Comment:
This will cause multiple events with same value, and we need to fix all usage places since some of them use !value instead of passed value.

Reply:
test

Reply:
test2

Reply:
Any thoughts?

Reply:
Previous `oldChecked` option wasn't always working correctly because it could contain not the latest state, when `isChecked` was changed by the `bind` property and not in the handler.
If we can update `oldChecked` in those cases too, that would probably be the best solution.

Reply:
Fixed in https://github.com/hcengineering/platform/pull/4509